src/pages/signup/index.tsx

```diff
@@ -1,5 +1,225 @@
+            />
+            <Button variant="contained" component="span" style={fileUploadStyle}>
+              Upload CV
+            </Button>
+          </label>
+
+          <label htmlFor="img-upload">
+            <input
+              accept="image/*"
+              id="img-upload"
+              type="file"
+              style={{ display: 'none' }}
+              onChange={handleFileUpload}
+            />
+            <Button variant="contained" component="span" style={fileUploadStyle}>
+              Upload Image
+            </Button>
+          </label>
+        </>
+      );
+    }
+    return null;
+  };
+
+  return (
+    <Grid container component="main" sx={gridStyle}>
+      <CssBaseline />
+      <Grid style={{ height: 'fit-content', width: 'fit-content', overflow: 'hidden' }} item xs={false} sm={4} md={6}>
+        <img src={imageSrc} alt="login" className="imageLogin" style={imageStyle} />
+      </Grid>
+      <Grid
+        item
+        xs={12}
+        sm={8}
+        md={6}
+        component={Paper}
+        elevation={6}
+        square
+        style={{ height: '100vh', overflow: 'auto', overflowX: 'hidden' }}
+      >
+        <Box sx={boxStyle}>
+          <Typography component="h1" variant="h5" sx={{ ml: 5, fontSize: '30px' }}>
+            Join us
+          </Typography>
+          <Box component="form" noValidate sx={{ mt: 5 }} onSubmit={formik.handleSubmit}>
+            <RadioGroup
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                justifyContent: 'center',
+                alignItems: 'center',
+              }}
+              aria-label="userType"
+              name="userType"
+              value={userType}
+              onChange={handleUserTypeChange}
+              row
+            >
+              <FormControlLabel
+                style={{ margin: '10px' }}
+                value="therapist"
+                control={<Radio />}
+                label="therapist"
+                labelPlacement="end"
+              />
+              <FormControlLabel
+                style={{ margin: '10px' }}
+                value="patient"
+                control={<Radio />}
+                label="patient"
+                labelPlacement="end"
+              />
+            </RadioGroup>
+            <TextField
+              margin="normal"
+              required
+              sx={textFieldStyle}
+              id="username"
+              name="username"
+              label="username"
+              autoFocus
+              onChange={formik.handleChange}
+              error={formik.touched.username && Boolean(formik.errors.username)}
+              helperText={formik.touched.username && formik.errors.username}
+            />
+            <TextField
+              margin="normal"
+              required
+              sx={textFieldStyle}
+              id="email"
+              name="email"
+              label="Email"
+              autoFocus
```

Comment:
```suggestion
```
AutoFoxus gonna led to focus last input which I find it a bit annoying, SO I'd prefer to delete this attribute for all inputs